src/DismissButton/DismissButton.js

```diff
@@ -1,6 +1,14 @@
 import React from "react";
 import PropTypes from "prop-types";
-import Icon from "react-hero-icon";
+// import Icon from "react-hero-icon";
+import {
+  XCircleIcon as XCircleIconSolid,
+  XIcon as XIconSolid,
+} from "@heroicons/react/solid";
+import {
+  XCircleIcon as XCircleIconOutline,
+  XIcon as XIconOutline,
+} from "@heroicons/react/outline";
```

Comment:
FYI: @sherakama @yvonnetangsu I'm starting the swap from "react-hero-icon" to "@heroicons/react" package in this PR and would love your thoughts on this implementation. I'm open to any suggestions you might have regarding how we want to differentiate between Solid and Outlined icons 😄

Reply:
This sounds good to me! 😄 

For HeroIcons that gets reused over and over, we have a custom component in SAA - might or might not want to do this for button icons or form icons?
https://github.com/SU-SWS/saa_alumni/blob/main/src/components/simple/heroIcon.js

Reply:
Ooo this would be super handy to have this reusable icon component in Decanter-React :D I'll roll that up in this PR as I continue to work through converting other components to use the new hero icon library.

Reply:
@mattanglin @adil-innovation-lab — tagging you both for feedback and suggestions on this as well! 😃